tests/content.spec.js

```diff
@@ -8,6 +8,16 @@ describe('Entity without declaration', () => {
     it('Entity should have passed content', () => {
         expect(render(<Bem block="Block">content</Bem>).text()).toBe('content');
     });
+
+    it('Entity via object should have passed content', () => {
+        expect(render(<Bem block={{ block : 'Block' }}>content</Bem>).text()).toBe('content');
+    });
+
+    it('Entity via function should have passed content', () => {
+        function Block () {}
+        Block.prototype.block = 'Block';
+        expect(render(<Bem block={Block}>content</Bem>).text()).toBe('content');
```

Comment:
why you need to do `<Bem block={Block}>` if you can just `<Block>`?

Reply:
for test coverage...

Reply:
Someone will generate nodes like these with generator from let's say bemjson and will shame us if these cases won't work.